extensions/data-workspace/package.vscode.json

```diff
@@ -1,4 +1,5 @@
 {
 	"name": "data-workspace-vscode",
-	"publisher": "ms-mssql"
+	"publisher": "ms-mssql",
+	"license": "https://raw.githubusercontent.com/Microsoft/azuredatastudio/main/extensions/LICENSE.txt"
```

Comment:
We can update the package.json license attribute with the new path. However, to maintain consistency, it would be preferable to override the value from the vscode.json file, as we do for other attributes.

Reply:
What consistency? I think having it be the same between both versions of the extension (unless they need to actually be different for some reason) would be best. e.g. just update the package.json values to point to the extension-local license file

Reply:
Also, since updating the license file, do we need to vbump the extensions versions? by having no code updates?